src/SFA.DAS.Assessor.Functions/OpportunityFinderDataSync.cs

```diff
@@ -28,7 +28,7 @@ public async Task Run([TimerTrigger("0 0 7 * * *", RunOnStartup = true)]TimerInf
                 log.LogInformation($"Update standard summary function started");
                 log.LogInformation($"Using api base address: {_assessorApiClient.BaseAddress()}");
 
-                await _assessorApiClient.UpdateStandardSummary();
+//                await _assessorApiClient.UpdateStandardSummary();
```

Comment:
And not commented out!

Reply:
That shouldn't have been checked in.  Good spot!